demo-transactions/src/main/kotlin/io/holixon/cqrshexagonaldemo/demoparent/transactions/domain/model/account/Account.kt

```diff
@@ -1,9 +1,21 @@
 package io.holixon.cqrshexagonaldemo.demoparent.transactions.domain.model.account
 
 import io.holixon.cqrshexagonaldemo.demoparent.transactions.domain.model.customer.CustomerNumber
+import java.math.BigDecimal
 
 data class Account(
-        var customerNumber: CustomerNumber,
-        var iban: Iban
+    var customerNumber: CustomerNumber,
+    var iban: Iban,
+    var balance: Money
+) {
+    fun deposit(amount: BigDecimal) {
```

Comment:
Just optional: You could define a value class for this amount instead of using BigDecimal directly. There you could ensure to deposit/withdraw only positive values.

Reply:
Nice tip 🙈! Thanks !